src/components/App.tsx

```diff
@@ -1615,7 +1617,6 @@ class App extends React.Component<AppProps, AppState> {
         ),
       );
     }
-    this.renderScene();
```

Comment:
The render flow has slighly changed because of this removal, simply as it would be moved from here to the end of the `App.componentDidUpdate`, could potentially break something.